.gitlab/pipeline/zombienet/cumulus.yml

```diff
@@ -30,6 +30,7 @@
     GH_DIR: "https://github.com/paritytech/cumulus/tree/${CI_COMMIT_SHORT_SHA}/zombienet/tests"
     LOCAL_DIR: "/builds/parity/mirrors/polkadot-sdk/cumulus/zombienet/tests"
     COL_IMAGE: "docker.io/paritypr/test-parachain:${DOCKER_IMAGES_VERSION}"
+    POLKADOT_PARACHIN_IMAGE: "docker.io/paritypr/polkadot-parachain-debug:${DOCKER_IMAGES_VERSION}"
```

Comment:
```suggestion
    POLKADOT_PARACHAIN_IMAGE: "docker.io/paritypr/polkadot-parachain-debug:${DOCKER_IMAGES_VERSION}"
```

Reply:
you probably also need to export it like other vars above

Reply:
also change the `needs` section, see https://github.com/paritytech/polkadot-sdk/pull/1386/files